packages/connectkit/src/components/Pages/PayWithToken/index.tsx

```diff
@@ -119,6 +128,16 @@ const PayWithToken: React.FC = () => {
         {selectedTokenOption && (
           <PaymentBreakdown paymentOption={selectedTokenOption} />
         )}
+        {payState === PayState.RequestingPayment && wcWallet && isMobile && (
+          <Button
+            icon={<ExternalLinkIcon />}
+            onClick={() => {
+              window.open(wcWallet.getWalletConnectDeeplink?.(""), "_blank");
+            }}
+          >
+            Pay with {wcWallet.name}
+          </Button>
```

Comment:
this might work better as an `a href` instead of button with `onClick` > works even when the window.open handler is missing like in the world app